.travis.yml

```diff
@@ -73,6 +73,11 @@ script:
       else
         ./node_modules/.bin/grunt jest:normal
       fi
+      echo 'Testing in server-render (HTML generation) mode...'
+      printf '\nmodule.exports.useCreateElement = false;\n' \
```

Comment:
I don't think you can escape stuff (newlines) inside echo reliably cross-platform.